lib/models/FamilyMember.dart

```diff
@@ -0,0 +1,13 @@
+import 'Family.dart';
+
+class FamilyMember {
+  final String memberID;
+  final String firstName;
+  final String middleName;
+  final String familyName;
+  final String dateOfBirth;
+  final String email;
+  final List<Family> mFamilies;
```

Comment:
```suggestion
  final List<Family> families;
```
Either is fine, but here we know its a member class, it will not be necessary extra information.